http/src/main/resources/init-resources/cluster-site-gce.conf

```diff
@@ -34,7 +34,7 @@
     # Append SameSite=None to cookies set by RStudio. This is required by some browsers because we
     # render RStudio in an iframe. There does not appear to be a way within RStudio to do this, hence
     # doing it in the proxy.
-    Header edit Set-Cookie ^(.*)$ $1;Secure;SameSite=None;HttpOnly "expr=%{REQUEST_URI} =~ m#/proxy/[^/]*/[^/]*/rstudio/.*#"
+    Header edit Set-Cookie ^(.*)$ $1;Secure;SameSite=None;HttpOnly;Partitioned "expr=%{REQUEST_URI} =~ m#/proxy/[^/]*/[^/]*/rstudio/.*#"
```

Comment:
I forgot we do this here. I wonder if it's still necessary, given that we also set `SameSite=None` in the Set-Cookie in the Leo proxy.

Reply:
Yeah I was not quite sure either, I can test without it, but thought it might be nice to keep it consistent